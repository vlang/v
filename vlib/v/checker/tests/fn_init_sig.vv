fn init() int {
	return 1
}

pub fn init() int {
	return 1
}
