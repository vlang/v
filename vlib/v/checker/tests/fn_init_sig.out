vlib/v/checker/tests/fn_init_sig.vv:1:1: error: fn `init` cannot have a return type
    1 | fn init() int {
      | ~~~~~~~~~~~~~
    2 |     return 1
    3 | }
vlib/v/checker/tests/fn_init_sig.vv:5:1: error: fn `init` must not be public
    3 | }
    4 | 
    5 | pub fn init() int {
      | ~~~~~~~~~~~~~~~~~
    6 |     return 1
    7 | }
vlib/v/checker/tests/fn_init_sig.vv:5:1: error: fn `init` cannot have a return type
    3 | }
    4 | 
    5 | pub fn init() int {
      | ~~~~~~~~~~~~~~~~~
    6 |     return 1
    7 | }
